File: Fortran/start-code/startcode.f90
program startcode
    implicit none
    character(30) :: name

    ! Begrüßung
    print *, 'Willkommen zu meinem Fortran-Programm!'

    ! Nutzereingabe
    print *, 'Bitte geben Sie Ihren Namen ein: '
    read *, name

    ! Begrüßung mit dem Namen
    print *, 'Hallo, ', trim(adjustl(name)), '! Schön, dass Sie hier sind.'
    print *, 'Das Programm wurde beendet.'
end program startcode

File: Lua/start-code/startcode.lua
-- Beschreibung: Ein einfaches Lua-Programm
-- Datum: 25.01.2025

-- Begrüßung
print("Willkommen zu meinem Lua-Programm!")

-- Funktion definieren
function begruessung()
    print("Das ist eine Beispiel-Funktion in Lua.")
end

-- Funktion aufrufen
begruessung()

-- Nutzereingabe
io.write("Bitte geben Sie Ihren Namen ein: ")
local name = io.read() -- Liest eine Eingabe von der Konsole

-- Begrüßung mit dem Namen
print("Hallo, " .. name .. "! Schön, dass Sie hier sind.")

-- Programm beenden
print("Das Programm wurde beendet.")
